pkg/lang/capabilities.go

```diff
@@ -52,64 +57,87 @@ func CompositePreprocessor(preprocessors ...CommentPreprocessor) CommentPreproce
 	}
 }
 
+func IsCLineCommentBlock(previous, current *sitter.Node) bool {
+	return previous != nil && current != nil &&
+		current.StartPoint().Row-previous.StartPoint().Row == 1 &&
+		strings.HasPrefix(current.Content(), "//") &&
+		strings.HasPrefix(previous.Content(), "//")
+}
+
+func IsNumberCommentBlock(previous, current *sitter.Node) bool {
```

Comment:
I was confused for a bit until I remembered `#` is called "number sign" sometimes. Considering the other function is `IsDoubleSlashCommentBlock` then perhaps we should name this one `IsPythonCommentBlock`? At least I'd use "hash" instead since I think that's the more common name for that character.

Alternatively, add a common implementation for the two like
```go
func CommentBlockByPrefix(prefix string) MergeCommentsPredicate {}
```
(function returning a function)

Reply:
> Considering the other function is IsDoubleSlashCommentBlock

It's called `IsCLineCommentBlock`.

> perhaps we should name this one IsPythonCommentBlock? At least I'd use "hash" instead since I think that's the more common name for that character.

"Number" is just carried over from how VS Code refers to them. Hash is fine too.

Reply:
> It's called IsCLineCommentBlock.

Oops, I meant "is not" 😄